api/v1alpha1/user_types.go

```diff
@@ -0,0 +1,62 @@
+/*
+RabbitMQ Messaging Topology Kubernetes Operator
+Copyright 2021 VMware, Inc.
+
+This product is licensed to you under the Mozilla Public License 2.0 license (the "License").  You may not use this product except in compliance with the Mozilla 2.0 License.
+
+This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// UserSpec defines the desired state of User
+type UserSpec struct {
+	// +kubebuilder:validation:Required
+	Name string    `json:"name"`
+	Tags []UserTag `json:"tags,omitempty"`
```

Comment:
if you leave []UserTag empty, what would happen?

Reply:
You get a user that has signalling access (e.g. AMQP) but no management access:
![image](https://user-images.githubusercontent.com/23215294/110306898-43bbde00-7ff6-11eb-91e5-10618d2a2ff1.png)

(Corresponds to `(None)` in [this table](https://www.rabbitmq.com/management.html#permissions))